README.md

```diff
@@ -64,6 +64,17 @@ To get started, create a codespace for this repository by clicking this 👇
 
 [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/AntonOsika/gpt-engineer/codespaces)
 
+## Getting Started with Docker
+
+**Building the image**:
+- `git clone https://github.com/AntonOsika/gpt-engineer.git`
+- `docker build --rm -t gpt-engineer .`
+
+**Running the image**:
+- `docker run -it --rm -e OPENAI_API_KEY="YOUR OPENAI KEY" -v ./your-project:/project gpt-engineer`
```

Comment:
the files that are created from container within volume are written as root user
```
drwxr-xr-x 4 root      root      4096 Jul 16 13:58 workspace
```
this make it's harder to modify, delete or execute files after creating. I would suggest to use this on Linux
```suggestion
- `docker run -it --rm -e OPENAI_API_KEY="YOUR OPENAI KEY" -u $(id -u):$(id -g) -v ./your-project:/project gpt-engineer`
```

Reply:
and when creating own project the files must be open for any user and group so that root can write
```
chmod a+rw -R my-project
```